indra/assemblers/html/assembler.py

```diff
@@ -106,6 +111,27 @@ class HtmlAssembler(object):
         the configuration entry indra.config.get_config('INDRA_DB_REST_URL').
         If None, the URLs are constructed as relative links.
         Default: None
+    sort_by : str or function or None
+        If str, it indicates which parameter in metric_dict to sort by, or
```

Comment:
It's not clear when reading this what metric_dict is as it is not a parameter or an attribute.

Reply:
Good catch, I think this bit of doc string got out of date.